assets/js/modules/analytics/components/setup/SetupFormGA4.stories.js

```diff
@@ -37,12 +37,20 @@ function Template() {
 
 export const WithoutExistingTag = Template.bind( null );
 WithoutExistingTag.storyName = 'Without Existing Tag';
+WithoutExistingTag.scenario = {
+	label: 'Modules/Analytics/Setup/SetupFormGA4/WithoutExistingTag',
+	delay: 250,
+};
 
 export const WithGA4Tag = Template.bind( null );
-WithGA4Tag.storyName = 'With GA4 Tag';
+WithGA4Tag.storyName = 'With GA4 Tag, non-matching property selected';
```

Comment:
Although the IB talks only about _adding_ stories, there are a number of existing stories which cover some or all of the AC and needed to be modified rather than adding new ones.

As such, the story names used have been kept consistent with existing stories.